pybikes/gbfs.py

```diff
@@ -17,6 +17,14 @@
     unicode = str
 
 
+def get_text(text):
+    if isinstance(text, str):
+        return text
+    if isinstance(text, list):
+        return next(iter(text))['text']
+    return ""
```

Comment:
I prefer this function to return `None` and cascade a failure than hiding the error with an empty string.

```suggestion
    return None
```